cglib/src/main/java/net/sf/cglib/core/Customizer2.java

```diff
@@ -0,0 +1,24 @@
+package net.sf.cglib.core;
+
+import org.objectweb.asm.Type;
+
+/**
+ * Customizes key types for {@link KeyFactory} right in constructor.
+ * This allows resolv
+ */
+public interface Customizer2 extends Customizer {
```

Comment:
-100 to this class.  Please figure out a way to make it work without introducing a file name Foo2.  We aren't the eclipse API.